### DISCUSSION TITLE
Motor Driver

### DISCUSSION THREAD

**Discussion by nledevil** [StuckAtPrototype/Racer#12] [General]
I am starting to add LCSC part numbers to the schematic so I can have jcbpcb create/populate the board. I noticed you are using Allegra A3908 brushed dc motor drivers. Is there a reason for this? A3908's are pretty pricey and a Texas Instruments motor driver would be significantly cheaper. Would there be any drawback to using a cheaper driver to make the overall cost per unit lower?

**Comment by StuckAtPrototype** [comment]
if it fits the footprint, then it could be a drop in replacement! it probably would be perfectly fine to go with a cheaper alternative

**Comment by andistorm** [comment]
Hey @nledevil,

> I am starting to add LCSC part numbers to the schematic

I would really appreciate if you could contribute this as PR
